src/errors.rs

```diff
@@ -56,6 +57,20 @@ quick_error! {
             description(err.description())
             display("protobuf error {:?}", err)
         }
```

Comment:
Can we remove this error?

Reply:
Maybe later. There's a lot of `?;` codes raising protobuf errors.

Reply:
Hmm, where do the protobuf errors come from if we're not using protobuf? Seems fine to leave it if that is the case though

Reply:
> where do the protobuf errors come

Existing codes calling protobuf APIs (which are redirected to the wrapper). Once we've completely eliminate them we can remove `Codec`!